chapters/errors.Rmd

```diff
@@ -758,7 +758,7 @@ can save us and our users a lot of needless work.
 ## Exercises {#errors-exercises}
 
 This chapter suggested several edits to `collate.py`.
-If our script now reads as follows:
+Suppose our script now reads as follows:
```

Comment:
Starting the sentence with 'If' with no following clause doesn't quite make sense. I've used "Suppose" instead but this could also be "Our updated script...". Happy to change as per review.